src/terminal/terminal-handlers.c

```diff
@@ -58,7 +58,12 @@
  * @param term
  *     The guac_terminal whose cursor should be advanced to the next row.
  */
-static void guac_terminal_linefeed(guac_terminal* term) {
+static void guac_terminal_linefeed(guac_terminal* term, bool force_wrap) {
+
+    /* Assign in column out of screen: 1 to avoid \n in clipboard or 0 to add \n */
+    guac_terminal_buffer_row* buffer_row = 
```

Comment:
The fact that this structure is being modified is not obvious from the function documentation - it should be documented there.

Reply:
Documentation updated.